server/db/models/User.js

```diff
@@ -84,7 +85,17 @@ User.authenticate = async function ({ username, password }) {
 User.findByToken = async function (token) {
   try {
     const { id } = await jwt.verify(token, process.env.JWT);
-    const user = User.findByPk(id);
+    const user = User.findOne({
+      where: {
+        id,
+      },
+      include: [
+        {
+          model: Order,
+          where: { status: "CREATED" },
+        },
+      ],
+    });
```

Comment:
when a user logs in, the orders associated with them will be sent back as part of the response data